mangum/handlers/utils.py

```diff
@@ -71,12 +71,14 @@ def handle_multi_value_headers(
 
 
 def handle_base64_response_body(
-    body: bytes, headers: Dict[str, str]
+    body: bytes,
+    headers: Dict[str, str],
+    text_mime_types: Optional[List[str]] = None,
 ) -> Tuple[str, bool]:
     is_base64_encoded = False
     output_body = ""
     if body != b"":
-        for text_mime_type in DEFAULT_TEXT_MIME_TYPES:
+        for text_mime_type in text_mime_types or DEFAULT_TEXT_MIME_TYPES:
```

Comment:
The default is already expected to be set by the constructor in the adapter module, so I think this should accept a required `text_mime_types` argument (informed by the config of the handler) and drop the `or DEFAULT_TEXT_MIME_TYPES` part.

Reply:
Done. My concern was backwards compatibility, but since mangum's major version is still 0 I guess it's OK. This would break my production code (and may do so for some people too), but the lesson here is to always pin your version until it's officially released (>=1.0.0).

Reply:
That’s a reasonable concern, and I typically try to at least do some kind of warning before breaking compatibility (even pre-1.0).

Not sure best way to do so in this case, any ideas? 

Otherwise, I think a minor version bump for this particular case should be okay.

Reply:
Good to know that's an option, but let's just go with minor version bump in this case.